Ray.jl/docs/make.jl

```diff
@@ -0,0 +1,13 @@
+using Ray
+using Documenter
+
+makedocs(; modules=[Ray],
+         sitename="Ray.jl",
+         authors="Beacon Biosignals",
+         pages=["API Documentation" => "index.md"],
+         strict=true)
+
+deploydocs(; repo="github.com/beacon-biosignals/ray_core_worker_julia_jll.jl.git", push_preview=true,
+           devbranch="main", dirname="Ray",
+           versions=["stable" => "Ray-v^", "Ray-v#.#",
+                     "dev" => "dev"])
```

Comment:
```suggestion
                     "dev" => "main"])
```
I assume?

Reply:
it's the sub-directory where dev docs are built (only have to specify this if we wanna deploy docs in this sub-package).  added a breadcrumb to change these settings when we move to own repo